package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "lint": "eslint packages tests --cache --fix --ext=js --ext=ts",
-    "pretest": "npm run lint --silent && npm run typecheck",
```

Comment:
Sure?

Reply:
`dataloader` and `locale` fail with pretest and I removed that for other test but forgot